CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## v1.12.0 (03.03.2021)
+
+### Enhancements
+
+#### :gear: Server-Side Web Assets Caching ([testcafe-hammerhead/#863](https://github.com/DevExpress/testcafe-hammerhead/issues/863))
+
+TestCafe's reverse-proxy can now cache web assets (like images, scripts and videos). When TestCafe revisits a website, it loads assets from this cache, saving time on repetetive network requests.
```

Comment:
Let's simplify this:
`TestCafe can now cache...`

Reply:
This part is important because it tells users that caching is performed server-side (not in the browser). How about this

```suggestion
TestCafe's proxy server can now cache web assets (like images, scripts and videos). When TestCafe revisits a website, it loads assets from this cache, saving time on repetitive network requests.
```

Reply:
Ok